docs-2.0/graph-computing/algorithm-description.md

```diff
@@ -16,6 +16,14 @@ Nebula Graph supports some graph computing tools. This topic describes the algor
     Different graph computing tools support different algorithms and different parameters. See below for details.
 -->
 
+!!! note
+
+    The algorithm parameters need to be set when performing graph computing, and the data source has requirements. The data source needs to contain starting vertexes and destination vertexes. PageRank, DegreeWithTime, SSSP, APSP, LPA, HANP, and Louvain algorithms must include weight.
```

Comment:
and the data source has requirements ---> and there are requirements for data sources

Reply:
vertexes--->vertices
starting vertexes ---> source vertices